Loenn/entities/dream_blocks/dream_move_block.lua

```diff
@@ -43,11 +67,28 @@ dreamMoveBlock.placements = {
             quickDestroy = false,
             direction = "Right",
             moveSpeed = 60.0,
-            noCollide = false
+            noCollide = false,
+			canSteer = false,
+			idleButtonsColor = "FFFFFF",
+			movingButtonsColor = "FFFFFF",
+			idleArrowColor = "FFFFFF",
+			movingArrowColor = "FFFFFF",
+			idleWobbleLinesColor = "FFFFFF",
+			movingWobbleLinesColor = "FFFFFF",
+			breakingWobbleLinesColor = "FFFFFF",
+			breakingCrossColor = "FFFFFF",
```

Comment:
indentation
```suggestion
            noCollide = false,
	    canSteer = false,
	    idleButtonsColor = "FFFFFF",
	    movingButtonsColor = "FFFFFF",
	    idleArrowColor = "FFFFFF",
	    movingArrowColor = "FFFFFF",
	    idleWobbleLinesColor = "FFFFFF",
	    movingWobbleLinesColor = "FFFFFF",
	    breakingWobbleLinesColor = "FFFFFF",
	    breakingCrossColor = "FFFFFF",
```

Reply:
Oh I didn't pay attention that the indentation was done with blank spaces. I used tabs which, in my editor didn't show any difference but looks like it did in your own. I'm replacing tabs with 4 spaces then.